packages/okta-react/src/Auth.js

```diff
@@ -53,7 +53,11 @@ export default class Auth {
   async isAuthenticated() {
     // If there could be tokens in the url
     if (location && location.hash && containsAuthTokens.test(location.hash)) return null;
-    return !!(await this.getAccessToken()) || !!(await this.getIdToken());
+    try {
+      return !!(await this.getAccessToken()) || !!(await this.getIdToken());
```

Comment:
Curious why we're trying to catch errors here, but neglecting the root cause of the error at the [source: `getAccessToken` and `getIdToken` methods](https://github.com/okta/okta-oidc-js/blob/e0d60e45a75d35936b1bf92af4cbe65c87cba648/packages/okta-react/src/Auth.js#L82).

Since the error is thrown during AuthJS's [`refresh`](https://github.com/okta/okta-auth-js/blob/master/lib/TokenManager.js#L127-L135) call due to the `prompt=none` param, we should probably add the `try catch` logic to the getter methods.

Then, users of the `getAccessToken` and `getIdToken` methods don't need to wrap it in a `try catch`, as the contract is:

- If you receive a token, it is valid
- If you do not receive a token, your session has ended.

Reply:
The idea was to handle the exception for the `isAuthenticated` function because the contract of that function is that returns true when you are authenticated and false otherwise, and it is used for routing. 
While for the other functions, let the error be visible to the developer, so they know why the `getAccessToken` is not giving a token even if it is available in the localStorage.

However, looking at the contract you posted above, makes sense what you are saying, because anyway the token in localStorage would not be valid when this exception is thrown. Will update!